src/test/java/com/jitterted/mobreg/adapter/in/web/member/EnsembleSummaryViewTest.java

```diff
@@ -32,9 +34,10 @@ public void withAnotherAcceptedMemberThenMemberAcceptedIsFalse() throws Exceptio
         MemberId memberId = MemberId.of(7L);
         member.setId(memberId);
         ensemble.acceptedBy(memberId);
+        List<Member> allRegisteredMembers = List.of(member, MemberFactory.createMember(5L, "other", "other_github"));
 
         EnsembleSummaryView ensembleSummaryView = EnsembleSummaryView
-                .toView(ensemble, MemberId.of(5L));
+            .toView(ensemble, MemberId.of(5L), allRegisteredMembers);
```

Comment:
Use of `allRegisteredMembers` here doesn't change any assertions, so why not `List.of(member)`?

Reply:
Ended up switching to TestMemberBuilder, so this is probably no longer relevant. Mostly reverted to your old tests in this class, but also switched to TestMemberBuilder  from MemberBuilder.